### ISSUE TITLE
Add blog for tree plantation event

### ISSUE THREAD

**Issue by czgdp1807** [rtrclubiitj/rtrclubiitj.github.io#2]
**Describe the bug/feature**

Blog entry for tree plantation has to be added

**To Reproduce(for bug)**

**Screenshots(for bug)**

**Device Details(for bug)**

**Other Comments**

The following content has to be added.
Tree Plantation Drive, IIT Jodhpur
By Sanskriti Hedaoo

“Planting a tree equals saving lives for free”

To beat the scorching heat and to beautify our campus, the Rotaract club organised a tree plantation drive on the IITJ Institute Foundation Day. The event was inaugurated on 2nd August by our respected Director Sir, Prof. Shantanu Chaudhary with the kind participation of Dr Shekhar Mande, Director Gen, CSIR, and Dr R Chidambaram, Chairman of Board of Governers, IITJ, who all planted the first plants of the drive.

The program was further extended with enthusiasm on 3rd August and started by 8:30 am, where faculty members and students showed up with great eagerness. The drive was aided by the Padmini Rotary of Jodhpur under its Green Mission. They provided us plants of over 100 different varieties which were planted across the campus. Students and faculty members also adopted plants and saplings to take care of. 

To add to excitement of the event, the raingods let down a heavy rainshower.. Despite the unexpected rains, everyone was dedicated and willing to complete the task and also enjoyed themselves. It was a successful and memorable program of the institute.

**Comment by czgdp1807** [comment]
@Saptashrungi Will you be able to make a PR for it?

**Comment by Saptashrungi** [comment]
What is a PR? I need info about it.

**Comment by czgdp1807** [comment]
PR -> Pull Request

**Comment by czgdp1807** [comment]
Refer, https://help.github.com/en/articles/creating-a-pull-request and https://help.github.com/en/articles/creating-a-pull-request-from-a-fork.
Have you forked this repository. If not, then please click the `Fork` button at the top right corner of this page.

**Comment by Saptashrungi** [comment]
Will do soon.. In lab now. Can you please give me a deadline for PR?

**Comment by czgdp1807** [comment]
No deadlines. However, if you will not do it by tonight, around 8:00 PM IST then I will make a PR. I will suggest you to give it a try.

**Comment by Saptashrungi** [comment]
Yes you are right.. That's how I will learn..I will do it soon:)

**Comment by czgdp1807** [comment]
I have mentioned the details at https://gitter.im/rtrclubiitj2019/website-maintain which created problem for you. Some changes are needed in the method of deployment. I will do that probably by tomorrow afternoon. After that the workflow will be setup.

**Comment by czgdp1807** [comment]
Please do not try to fix this issue for now.
